scripts/vcpkg_ports/opencascade/vcpkg.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "opencascade",
+  "version": "7.8.0",
+  "description": "Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.",
+  "homepage": "https://github.com/Open-Cascade-SAS/OCCT",
+  "license": "LGPL-2.1-only",
+  "supports": "!xbox",
+  "dependencies": [
+    {
+      "name": "angle",
+      "platform": "uwp"
+    },
+    {
+      "name": "opengl",
+      "platform": "!(android | ios | uwp | wasm32)"
+    },
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ],
+  "default-features": [],
```

Comment:
Note: I copied the `opencascade` portfile from `vcpkg` to remove the `freetype` default-feature.
More direct ways to do this would involve updates to uberenv to have a project package file.

Compare to:
https://github.com/microsoft/vcpkg/blob/98ffd29b9e6a07db04a0899d1be351e9e3bd1e54/ports/opencascade/vcpkg.json#L27-L32